Automattic/jetpack
# Remove ModuleOptionEnum and ModuleOptionTextInput components

Fixes #4386 #4383 #4378 #4379

#### Changes proposed in this Pull Request:

* Adds component `ModuleSettingsRadio` which replaces `ModuleOptionsEnum` previously defined in `throwme.jsx`
* Makes the **Post By Email** _Regenerate Address_ setting work.
* Removes the `throwme.jsx`.

#### Testing instructions

1. Update one of the radio button options in the **Carousel** module (Under the Appearance tab).
2. Refresh the page and check the option change persisted.
3. In the **Post By Email** settings (Under the Writing tab), check that the _Regenerate address_ button works by updating the email address in the text input on the left.
